app/controllers/proposal/collection_draft_proposals_controller.rb

```diff
@@ -4,7 +4,7 @@ class CollectionDraftProposalsController < CollectionDraftsController
 
     # TODO: Limit this to only the things a user is supposed to do, also need a new one for approver
     # TODO: Also need one for functions which both can perform (e.g. show)
-    before_action :ensure_non_nasa_draft_user
+    before_action :ensure_non_nasa_draft_permissions
```

Comment:
I think there is value in having a user_permissions and approver_permissions before action.  Specifically, these before actions can be used to ensure that users do not have access to approve, for instance.  Also, approvers do not need to be able to CRUD proposals.

Reply:
I do not disagree. However, those are not yet needed and without clearly knowing how they will be used I would rather do what is necessary for now, knowing it might need to change as we add functionality - which is why I left your comments above and added comments to the tests' configuration_helpers file.